cmake/Packaging.cmake

```
@@ -98,16 +101,90 @@ endmacro()
 #
 # Linux packages
 #
+
+macro(configure_linux_package_name)
+  ##Get Distro name for fall back
+  execute_process(
+    COMMAND bash "-c" "cat /etc/os-release | grep ^ID= | sed 's/ID=//g'"
+    OUTPUT_VARIABLE _DISTRO_NAME
+    OUTPUT_STRIP_TRAILING_WHITESPACE)
+  string(REPLACE "\"" "" DISTRO_NAME "${_DISTRO_NAME}")
+  message(STATUS "DISTRO NAME: ${DISTRO_NAME}")
+
+  execute_process(
+    COMMAND bash "-c"
+            "cat /etc/os-release | grep ^ID_LIKE= | sed 's/ID_LIKE=//g'"
+    OUTPUT_VARIABLE _DISTRO_LIKE
+    OUTPUT_STRIP_TRAILING_WHITESPACE)
+  string(REPLACE "\"" "" DISTRO_LIKE "${_DISTRO_LIKE}")
+  message(STATUS "DISTRO LIKE: ${DISTRO_LIKE}")
+
+  execute_process(
+    COMMAND
+      bash "-c"
+      "cat /etc/os-release | grep ^VERSION_CODENAME= | sed 's/VERSION_CODENAME=//g'"
+    OUTPUT_VARIABLE _DISTRO_CODENAME
+    OUTPUT_STRIP_TRAILING_WHITESPACE)
+  string(REPLACE "\"" "" DISTRO_CODENAME "${_DISTRO_CODENAME}")
+  message(STATUS "DISTRO CODENAME: ${DISTRO_CODENAME}")
+
+  execute_process(
+    COMMAND bash "-c"
+            "cat /etc/os-release | grep ^VERSION_ID= | sed 's/VERSION_ID=//g'"
+    OUTPUT_VARIABLE _DISTRO_VERSION_ID
+    OUTPUT_STRIP_TRAILING_WHITESPACE)
+  string(REPLACE "\"" "" DISTRO_VERSION_ID "${_DISTRO_VERSION_ID}")
+  message(STATUS "DISTRO ID: ${DISTRO_VERSION_ID}")
+
+  #CHECK IF RUNNING ON GITHUB FOR A RUN ATTEMPT TO USE IT
```

```suggestion
  #CHECK IF RUNNING ON GITHUB FOR A RUN ATTEMPT TO USE IT
  # Check if it is running on GitHub and start the counter
```

Not sure I understand this comment, perhaps you could reword it? Maybe add some context as to why this is needed.

---

It does not start a counter this var is set by github when the job starts. 
Its how many Attempts the job has had , in packages it the -# before the distro name 
`-1~jammy`  for example